.github/workflows/deployment.yml

```diff
@@ -0,0 +1,119 @@
+name: Deployment
+
+concurrency: 
+  group: production
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+      go_version:
+        default: "1.19"
+        type: string
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ${{ inputs.go_version }}
+      - name: Configure GoReleaser
+        run: |
+          touch CHANGELOG.md
+          sed '/- id: windows$/,/^$/d; /- id: macos$/,/^$/d' .goreleaser.yml >.goreleaser.generated.yml
```

Comment:
Rather than do this would it make more sense to have three goreleaser files checked in, one for each platform?

Reply:
I'm going to clean this approach up, but I still prefer having a single “master” Goreleaser config instead of syncing changes between multiple of them. Furthermore, a single Goreleaser config makes it easy to test cross-compilation for all platforms at once locally